eng/native/init-os-and-arch.sh

```diff
@@ -74,7 +74,7 @@ case "$CPUName" in
 	arch=ppc64le
 	;;
     *)
-        echo "Unknown CPU $CPUName detected, configuring as if for x64"
-        arch=x64
+        echo "Unknown CPU $CPUName detected!"
```

Comment:
We can use the same approach on line 16 for operating system.

Reply:
OK, I had updated.
Thanks